### PR TITLE
chore: update presigned canary to use api endpoint instead of rest

### PR DESCRIPTION
**Pull request by krispraws**

### CODE REVIEW

**Review by PDXKimani — APPROVED** [review]

File: tests/momento_signer/test_momento_signer.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import jwt
 import pytest
 from jwt.api_jwk import PyJWK
-from momento_signer import CacheOperation, MomentoSigner, SigningRequest
 
+from momento_signer import CacheOperation, MomentoSigner, SigningRequest
```
Comment (nidhidhamnani) [inline tests/momento_signer/test_momento_signer.py:5]: Don't know what changed here

File: src/momento_signer/momento_signer.py [hunk 2]
```diff
@@ -130,15 +130,12 @@ def create_presigned_url(self, hostname: str, signing_request: SigningRequest) -
         cache_name = quote(signing_request.cache_name(), safe="")
         cache_key = quote(signing_request.cache_key(), safe="")
         if signing_request.cache_operation() == CacheOperation.GET:
-            return f"https://rest.{hostname}/cache/get/{cache_name}/{cache_key}?token={token}"
+            return f"https://api.{hostname}/cache/{cache_name}/{cache_key}?token={token}"
```
Comment (nidhidhamnani) [inline src/momento_signer/momento_signer.py:133]: looks good

**Review by nidhidhamnani — APPROVED** [review]

File: tests/momento_signer/test_momento_signer.py (hunk 1 above)
Comment (krispraws) [reply tests/momento_signer/test_momento_signer.py:5]: No idea, I didn't touch the file. A newline got added somehow.

**Review by cprice404 — APPROVED** [review]
cool

**Pull request merged**